src/main/java/org/gridsuite/actions/server/ContingencyListController.java

```diff
@@ -98,12 +98,23 @@ public ResponseEntity<List<Contingency>> exportContingencyList(@PathVariable("id
     @Operation(summary = "Create a script contingency list")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The script contingency list have been created successfully")})
     public ResponseEntity<ScriptContingencyList> createScriptContingencyList(@RequestParam(required = false, value = "id") UUID id,
-                                                                             @RequestBody(required = false) ScriptContingencyList script) {
```

Comment:
Weirdly enough without that changes RestController considers there is an ambiguity between this endpoint and the new one I added in order to duplicate a script contingency list.
I spend a good amount of time trying to figure out why there was an ambiguity in spite of the parameters difference and the only fix I found was removing this section. I figured the change would not  negatively the system since without a script in the body the method can't do much.
I'm open to suggestions on this one.